src/utils/uid.ts

```diff
@@ -3,5 +3,10 @@ function hexify(number: number) {
 }
 
 export function generateUID() {
+    // TODO: remove any cast once types are updated
+    if ((crypto as any).randomUUID) {
+        return (crypto as any).randomUUID().replaceAll('-', ' ');
```

Comment:
Did you mean to replace "-" with "" (remove dashes)?

Reply:
Yes, I did. I'll add unit tests for UID utils.